src/main/kotlin/com/nextstep/jngcii/step2/main.kt

```diff
@@ -0,0 +1,23 @@
+package com.nextstep.jngcii.step2
+
+import java.util.Scanner
+
+fun main(args: Array<String>) {
```

Comment:
```suggestion
import java.util.Scanner

fun main() {
```
낮은 코틀린 버전에서는 args 명시가 필수적이였지만 이제는 제거해도 됩니다!